src/main/webapp/WEB-INF/spring/appServlet/servlet-context.xml

```diff
@@ -48,7 +48,8 @@
     <beans:property name="nedUrl" value="${NED_URL:}" />
     <beans:property name="casUrl" value="${CAS_URL:}" />
     <beans:property name="themePath" value="${THEME_PATH:}" />
-    <beans:property name="environment" value="${ENVIRONMENT:prod}" />
+    <beans:property name="environment" value="${ENVIRONMENT:dev}" />
+    <beans:property name="rootRedirect" value="${ROOT_REDIRECT:https://www.plos.org/}" />
```

Comment:
The canonical URL omits the `www` since the marketing site went to wordpress.

Reply:
Good to know! I'm still waiting on the correct url from marketing; when I get it I'll update this.